fe/fe-core/src/main/java/com/starrocks/load/routineload/RoutineLoadMgr.java

```diff
@@ -620,6 +623,25 @@ public void replayChangeRoutineLoadJob(RoutineLoadOperation operation) {
                 .build());
     }
 
+    public void setRoutineLoadJobOtherMsg(String reason, TxnCommitAttachment txnCommitAttachment) {
+        if (!(txnCommitAttachment instanceof RLTaskTxnCommitAttachment)) {
+            return;
+        }
+        RLTaskTxnCommitAttachment rLTaskTxnCommitAttachment = (RLTaskTxnCommitAttachment) txnCommitAttachment;
+        long jobId = rLTaskTxnCommitAttachment.getJobId();
+        if (jobId == 0) {
+            return;
+        }
+        RoutineLoadJob routineLoadJob = getJob(jobId);
+        if (routineLoadJob != null && reason != null) {
+            String otherMsg = String.format("The %s task have complained: %s at %s",
+                    DebugUtil.printId(rLTaskTxnCommitAttachment.getTaskId()),
+                    reason,
+                    new Date(rLTaskTxnCommitAttachment.getCreationTimestampMs()));
```

Comment:
The new version would add a timestamp to setOtherMsg automatically. Is this necessary?